### PR TITLE
Only show first tag - put how to link to the top

### PR DESCRIPTION
**Pull request by alexsuter**
grafik

### CODE REVIEW

**Comment by alexsuter** [comment]
do you have a better idea?

File: src/app/pages/market/market.twig [hunk 1]
```diff
@@ -64,20 +64,22 @@
...
     </p>
 
+  
+
     <div class="row">
       <div class="6u 12u(small)" style="margin-bottom:10px;">
         <span class="dropdown">
           <label>Topic</label>
           <select id="tags" multiple="multiple">
-            {% for tag in tags %}
```
Comment (ivy-lli) [inline src/app/pages/market/market.twig:78]: Isn't this one the topic/tag select filter? Why did you remove this one?

**Review by ivy-lli — APPROVED** [review]
Totally fine for me, but I think you break the topic/tag filter select.

File: src/app/pages/market/market.twig (hunk 1 above)
Comment (alexsuter) [reply src/app/pages/market/market.twig:78]: ups
